stf_functional_tests.yml

```diff
@@ -13,6 +13,28 @@
         group: wheel
         state: link
 
+- name: "Set up defaults"
+  hosts: overcloud_nodes
+  tags: always
+  tasks:
+    - include_vars:
+        file: defaults/main.yml
```

Comment:
Do you need to include the default vars? Aren't they available by default?

Reply:
Somehow, no, and I'm not certain why, but this was my workaround.

Reply:
Hrmmm might be worth an issue to investigate why that is, because I'm a bit concerned we may run into some oddities that are hard to debug if this is necessary. Or at least understand why it's necessary and correct.

Reply:
I opened issue https://github.com/infrawatch/functional-tests/issues/8, I think it's dues to the layout of the directory, and ansible not recognising it as a single role

Reply:
I fixed that, little mix up on my part with plays vs roles, and how this was originally set up. This is what happens when you write your first ansible scripts and they actually get used.